src/Context/authorization-context.js

```diff
@@ -0,0 +1,85 @@
+import {
+    createContext,
+    useContext,
+    useEffect,
+    useReducer
+} from "react";
+
+const AuthContext = createContext(null);
+
+const authReducer = (authState, action) => {
```

Comment:
You can put this reducer function in a separate file.

Reply:
Yep, will refractor the code later.